staging/src/k8s.io/metrics/pkg/client/clientset_generated/clientset/typed/metrics/v1alpha1/podmetrics.go

```diff
@@ -80,8 +79,8 @@ func (c *podMetricses) List(opts v1.ListOptions) (result *v1alpha1.PodMetricsLis
 
 // Watch returns a watch.Interface that watches the requested podMetricses.
 func (c *podMetricses) Watch(opts v1.ListOptions) (watch.Interface, error) {
+	opts.Watch = true
 	return c.client.Get().
-		Prefix("watch").
```

Comment:
> This change breaks compatibility with sufficiently old versions of kubernetes, I think? @deads2k fyi

Thanks for the heads up.  I think we supported the query param as far back as kube 1.0: https://github.com/kubernetes/kubernetes/blob/release-1.0/pkg/apiserver/resthandler.go#L251